create-entry.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+JEKYLL_ASSETS_DIR=$HOME'/Documents/portfolio/assets/'
+JEKYLL_POSTS_DIR=$HOME'/Documents/portfolio/_posts/'
+
+FOLDER_NAME='';
```

Comment:
Just going to point out that `SCREAMING_VARIABLES` aren't necessary in shell script; I'd rather read `$foo` instead of `$FOO`. I tend to use `ALLCAPS` just for constants.